<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorMappingChooser</class>
 <widget class="QDockWidget" name="ColorMappingChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color Mapping</string>
  </property>
  <widget class="QWidget" name="content">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="relatedRenderView">
      <property name="text">
       <string>&lt;b&gt;(No Render View selected)&lt;/b&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="scalarsGroupBox">
      <property name="title">
       <string>&amp;Scalars</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="componentLabel">
         <property name="text">
          <string>&amp;Component</string>
         </property>
         <property name="buddy">
          <cstring>componentSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="componentSpinBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="maxLabel">
         <property name="text">
          <string>&amp;Max</string>
         </property>
         <property name="buddy">
          <cstring>maxValueSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="maxValueSpinBox">
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="minLabel">
         <property name="text">
          <string>M&amp;in</string>
         </property>
         <property name="buddy">
          <cstring>minValueSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="minValueSpinBox"/>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QComboBox" name="scalarsComboBox"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gradientGroupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>&amp;Gradient</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QComboBox" name="gradientComboBox"/>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="nanColorButton">
         <property name="text">
          <string>&amp;NaN-Color</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="legendGroupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Color &amp;Legend</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="1">
        <widget class="QComboBox" name="legendPositionComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QCheckBox" name="legendAlignTitleCheckBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Align title with color bar</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Title &amp;Font Size</string>
         </property>
         <property name="buddy">
          <cstring>legendTitleFontSize</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="legendTitleFontSize"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&amp;Label Font Size</string>
         </property>
         <property name="buddy">
          <cstring>legendLabelFontSize</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="legendLabelFontSize"/>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QCheckBox" name="legendBackgroundCheckBox">
         <property name="text">
          <string>&amp;Opaque Background</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>&amp;Position</string>
         </property>
         <property name="buddy">
          <cstring>legendPositionComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>&amp;Title</string>
         </property>
         <property name="buddy">
          <cstring>legendTitleEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="legendTitleEdit">
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
